Mathlib/LinearAlgebra/BilinearMap.lean

```diff
@@ -383,6 +383,32 @@ theorem compr₂_apply (f : M →ₗ[R] Nₗ →ₗ[R] Pₗ) (g : Pₗ →ₗ[R]
     f.compr₂ g m n = g (f m n) := rfl
 #align linear_map.compr₂_apply LinearMap.compr₂_apply
 
+/-- Apply linear equivalences on the arguments of a bilinear map. -/
+def congrl₁₂ (e₁ : Mₗ ≃ₗ[R] Nₗ) (e₂ : Pₗ ≃ₗ[R] Qₗ) :
+    (Mₗ →ₗ[R] Pₗ →ₗ[R] M) ≃ₗ[R] (Nₗ →ₗ[R] Qₗ →ₗ[R] M) where
```

Comment:
I'm not convinced we need this at all, as it's just two applications of `LinearEquiv.arrowCongr`
```suggestion
/-- Apply linear equivalences on the arguments of a bilinear map. -/
def congrl₁₂ (e₁ : Mₗ ≃ₗ[R] Nₗ) (e₂ : Pₗ ≃ₗ[R] Qₗ) :
    (Mₗ →ₗ[R] Pₗ →ₗ[R] M) ≃ₗ[R] (Nₗ →ₗ[R] Qₗ →ₗ[R] M) :=
  e₁.arrowCongr <| e₂.arrowCongr <| .refl R M
```
(Note this is definitionally equal to what you wrote.) If you want, you can update the definition of `BilinForm.congr` to use `LinearEquiv.arrowCongr`